admin_manual/configuration_user/two_factor-auth.rst

```diff
@@ -5,57 +5,64 @@ Two-factor authentication
 =========================
 
 Two-factor authentication adds an additional layer of security to user accounts. In order to log
-in on an account with two-factor authentication (2FA) enabled, it is necessary to provide both the
-login password and another factor. 2FA in Nextcloud is pluggable, meaning that they are not part
-of the Nextcloud Server component but provided by featured and 3rd-party Nextcloud apps.
+in on an account when two-factor authentication (2FA) enabled, it is necessary to provide both the
```

Comment:
it is necessary to provide -> you must provide
(direct language is often easier to understand)